### ISSUE TITLE
Add customized question permanently

### ISSUE THREAD

**Issue by yangxu698** [surveyjs/survey-library#1930]
### Are you requesting a feature, reporting a bug or asking a question?
Asking a question

### What is the current behavior?
We are building hundreds of surveys, they have different question content, but we will need to use one question type(including color setting etc.) repeatedly, we can add to toolbar within one same survey, so it can be added or dragged to use.

### What is the expected behavior?
do you think if we can add it to the different surveys as a default question type, like Checkbox, html?

#### Provide the test code and the tested page URL (if applicable)

Tested page URL:

Test code

```
your_code_here

```

### Specify your

* browser: Chrome
* browser version: 78.0.3904.97
* surveyjs platform (angular or react or jquery or knockout or vue): jquery
* surveyjs version: 1.1.18

**Comment by tsv2013** [comment]
Is your question related to SurveyJS Creator?

**Comment by yangxu698** [comment]
Yes, what I want to implement is embedding a default question type when creating any new survey.could be the same type as like check box, but with certain content populated so it can be applied to different surveys quickly.
